docs/README.md

```diff
@@ -725,3 +750,30 @@ public static async handle(event: StockMoved, register: Register): Promise<void>
   ...
 }
 ```
+
+### 4. Entities and reducers
+
+#### Entities naming convention
+
+#### Creating entities
+
+#### The reducer function
+
+#### Aggregate data using entities
+
+#### Eventual consistency
+
+### 5. Read models and projections
```

Comment:
will we talk here about how we could project several entities into a read model?

Reply:
I think Javi said that aggregating in read-models isn't the right approach, and instead, we should have `Aggregated Entities` and then read model for those. 

@javiertoledo, Is that correct?

With that in mind, I added this heading [`Aggregate data using entities`](https://github.com/boostercloud/booster/pull/215/files#diff-e70100059cc3a7376740bbe9751a082eR762) under the entities section.

We can advise the users three things:

1. To aggregate data only in entities
1. do it only in read models
1. or, aggregate data in both

With the proposed approach, we go with the first choice

Reply:
There's not a firm decision on how we're going to design access from handlers to read models or if that makes sense at all. There's some discussion about this topic in [an internal Confluence page](https://agilemonkeys.atlassian.net/wiki/spaces/B/pages/1199341571/Fixing+Entity+ReadModel+and+GraphQL+overlap?focusedCommentId=1207173121#comment-1207173121)

Basically, Read Models are designed to optimize reads from the API side, but they're all public by default, so there are some use cases where we want to query the state from a handler but we can't publish the queried data as a read model in the public API. On the other side, it's important that we store the current state in a database that we can query to answer questions that require obtaining a list of entities and things like that. There are many alternatives:

1. Read models can be private, and we let handlers to read any read model
2. Read models are only accessible by the API and we implicitly store all Entities in tables and create a querying mechanism for entity tables (I bet we can reuse the current mechanism for Read Models). The good thing about this is that "it just works", bad thing is that we generate a lot of unnecessary lambda activity.
3. Read models are only for the API, but we add a cached=true parameter to the @Entity decorator that makes Booster generate a table for this entity and keep it up to date. Cached entities can be queried.

We have to evaluate pros and cons of each option to make sure that we pick a solution that makes sense, don't introduce weird extra config parameters to the developer and is cost-efficient

Reply:
Hopefully, we will have that clear when we reach that point in the docs, if not we can postpone the decision and document it later